RecoMET/METFilters/plugins/BadGlobalMuonTagger.cc

```diff
@@ -1,130 +0,0 @@
-#include "FWCore/Framework/interface/stream/EDFilter.h"
```

Comment:
why is this file removed?
It's still used in the bad muon mitigation machinery.
was it just dropped by mistake?